src/lib/Runner.ts

```diff
@@ -50,9 +50,16 @@ export class Runner {
         landRequest: landRequest.get(),
       });
 
+      const triggererUserMode = await permissionService.getPermissionForUser(
+        landRequest.triggererAaid,
+      );
+
       // TODO: Pass this commit in to isAllowed to land and make sure it hasnt changed
```

Comment:
Is the TODO still valid?

Reply:
Not sure, will have to ask Luke

Reply:
Yea, still valid.
The check here would be an edge case that would be super hard to hit.
Someone would have to start landing a PR, then push a change, hack in approvals, green builds, etc whilst landkid is still running, then recieve the green build for the previous state, all before the landkid build finished.

Would be pretty unlikely, but still technically doable